carbonmark-api/test/fixtures/tokens.ts

```diff
@@ -34,6 +34,7 @@ const prices: GetPoolPricesQuery["prices"] = [
 
 export const token = {
   symbol: "TCO2-VCS-1764-2020",
+  id: "1",
 };
```

Comment:
This token isn't an ERC1155 and thus won't have a tokenId.

Reply:
it's tricky because the source of this is called "token" in the subgraph, whereas the other comes from carbon credits but it refers to the tokenId.
I have moved it out in [645c800](https://github.com/KlimaDAO/klimadao/pull/2188/commits/645c800b440a1bb93ae34fdf5773725e6e9d8c10)

Reply:
this is now obsolete due to the subgraph changes where we now have tokenId as part of a listing